nservicebus/operations/metrics_reporting_metrics_[1.0,1.1).partial.md

```diff
@@ -0,0 +1,29 @@
+## Reporting metrics data
```

Comment:
Do we want to add note somewhere here about the fact that those methods are obsoleted and will be gone in 2.0?